### ISSUE TITLE
LinuxED doesn't support the Discord snap package

### ISSUE THREAD

**Issue by saltedcoffii** [Cr3atable/LinuxED#8]
Please support the Discord snap package in the install menu, possibly with another number option. Thank you!!

**Comment by twnlink** [comment]
I haven't had any issues with Snap Discord, though there may be a file permission error. I remember that when I made LED that it had issues with Snap not being able to access ED's files due to a permission error, though that may be an issue with Flatpak instead.

**Comment by saltedcoffii** [comment]
I haven't had a problem with the Flatpak.

**Comment by twnlink** [comment]
> I haven't had a problem with the Flatpak.

Got it. So when you posted this issue did you not see Snap in the list of installed clients when running LED or what?

**Comment by saltedcoffii** [comment]
No, I installed the package via snap and it didn't seem to work. LinuxED injected it just fine but due to snapcraft's sandboxing, Discord couldn't access Enhanced Discord's files in my Downloads directory, and returned a JavaScript error saying it couldn't find injection.js. Thanks for your help on this btw.
